include/crm/common/ipc_controld.h

```diff
@@ -37,6 +37,8 @@ enum pcmk_controld_api_reply {
     pcmk_controld_reply_nodes,
 };
 
+const char *pcmk__controld_api_reply2str(enum pcmk_controld_api_reply reply);
+
```

Comment:
If these are meant to be internal, they should go in ipc_internal.h (or new ipc_*_internal.h files); if they're meant to be public, single underbar

Reply:
whoops

Definitely private.